platforms/hyperledger-fabric/configuration/roles/helm_component/templates/ca-orderer.tpl

```diff
@@ -76,7 +76,7 @@ spec:
       servicetype: ClusterIP
       ports:
         tcp:
-          port: {{ component_services.ca.grpc.port }}
+          clusteripport: {{ component_services.ca.grpc.port }}
```

Comment:
so this was causing ca server to not run with proxy none?

Reply:
It is different, it should be 'clusterIPPort' instead of 'port'. The values in the image correctly point to the local 'values.yaml' file. However, the 'HelmRelease' file has 'port' defined instead of 'clusterIPPort'.

<img width="288" alt="image" src="https://github.com/hyperledger/bevel/assets/124864640/1580031f-d547-4c46-834c-0f3731478ae6">